src/wizard/monster-info-spoiler.cpp

```diff
@@ -178,9 +175,7 @@ SpoilerOutputResultType spoil_mon_info(concptr fname)
         return SpoilerOutputResultType::FILE_OPEN_FAILED;
     }
 
-    char title[200];
-    put_version(title);
-    spoil_out(std::string("Monster Spoilers for ").append(title).append("\n").data());
+    spoil_out(std::string("Monster Spoilers for ").append(get_version()).append("\n").data());
```

Comment:
Since spoil_out accepts std::string_view type, the last .data() is unnecessary.

Reply:
The forced push drops the call to data() on string passed to the two calls spoil_out().  Another change in the forced push is to add const to the declaration at main/angband-initializer.cpp:200.